exporters/BxDRobotExporter/BxDRobotExporter/BxDRobotExporter/StandardAddInServer.cs

```diff
@@ -55,16 +55,14 @@ public bool PendingChanges
 
         //Ribbon Pannels
         RibbonPanel SetupPanel;
-        RibbonPanel FilePanel;
         RibbonPanel SettingsPanel;
+        RibbonPanel FilePanel;
```

Comment:
By C# naming conventions, private fields are camel cased. Just something to keep in mind going forward.

Reply:
Diff groups have different conventions, to avoid refactoring all of the code, we should probably stick to the conventions currently used in the code, which is pascal case for private fields, and all caps for constants.